scripts/ai/PathfinderController.lua

```diff
@@ -305,6 +329,16 @@ function PathfinderController:findPathToGoal(context, goal, numRetries)
     return true
 end
 
+function PathfinderController:findAnalyticPathFromVehicleToGoal(goal, allowReverse)
```

Comment:
Could you add a description what this function does?

Reply:
Added

Reply:
Added